### PR TITLE
Upload a first draft of XSD

### PR DESCRIPTION
**Pull request by rsynek**

### CODE REVIEW

File: xsd/optaplanner/index.html.haml [hunk 1]
```diff
@@ -6,7 +6,7 @@ priority: 0.1
 ---
 %h1 #{page.title}
 %ul
-  - for file in Dir.entries("xsd").sort
+  - for file in Dir.entries("xsd/optaplanner").sort
```
Comment (rsynek) [inline xsd/optaplanner/index.html.haml:9]: Specifically moved the XSD to `xsd/optaplanner` to match its namespace:
https://www.optaplanner.org/xsd/optaplanner

Comment (ge0ffrey) [reply xsd/optaplanner/index.html.haml:9]: I presume that the links on the generated page in the _site output still work?

Comment (rsynek) [reply xsd/optaplanner/index.html.haml:9]: Are there any links pointing to this page? I haven't found any and accessed directly to the built website via URL

**Review by ge0ffrey — APPROVED** [review]

File: xsd/optaplanner/index.html.haml (hunk 1 above)
Comment (ge0ffrey) [reply xsd/optaplanner/index.html.haml:9]: Perfect :)

**Pull request merged**